modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -306,7 +306,12 @@ You can set a soft limit for a minimum free disk space alert. This soft limit ge
 
 NOTE: The alert threshold can be set in either bytes or percentage of total space. To disable one threshold in favor of the other, set it to zero.
 
-When a disk exceeds the set threshold, `redpanda_storage_disk_free_space_alert` updates, and an error message is written to the Redpanda service log.
+When a disk exceeds the configured threshold, Redpanda updates the xref:reference:public-metrics-reference.adoc#redpanda_storage_disk_free_space_alert[`redpanda_storage_disk_free_space_alert`] metric appropriately and writes an error level `storage space alert` message to the service log. The message looks like the following:
```

Comment:
```suggestion
When a disk exceeds the configured threshold, Redpanda updates the xref:reference:public-metrics-reference.adoc#redpanda_storage_disk_free_space_alert[`redpanda_storage_disk_free_space_alert`] metric and writes an error level `storage space alert` message to the service log. Example:
```

Reply:
Adjusts to which one of those states?
0-OK, 1-low space, 2-degraded.

Reply:
When does it hit low space?

Reply:
Hmm, debating whether to list that here or whether to put it in the metric page. Our metrics documentation needs a lot of work in general, may be better to put it there?

Reply:
I would put this info here and we can replicate it in the future. Honestly I think we can only answer the initial ticket ask if we answer those questions.

Reply:
+1 on @Deflaimun 's suggestion.